src/__test__/test-with-providers.tsx

```diff
@@ -42,6 +44,8 @@ const TestWithProviders = ({
   cellCoordMock = undefined,
   selectionDispatchMock = undefined, // Can be used to avoid selectionDispatch infinite loop
   direction = 'ltr',
+  rootElement = {} as HTMLElement,
+  embed = {} as stardust.Embed,
```

Comment:
casting to make sure they are not optional. We know they are, but acccording to the type it can be undefined, since we are sharing the type with the native solution. Will change this when we know the future of the native table in this repo